src/js/components/Carousel/__tests__/__snapshots__/Carousel-test.js.snap

```diff
@@ -203,7 +203,7 @@ exports[`Carousel basic 1`] = `
   border: none;
   padding: 0;
   text-align: inherit;
-  opacity: 0.3;
+  color: #666666;
```

Comment:
This change (and the other similar ones) is an indication of a potential backward compatibility issue.

Reply:
From my understanding, we want to change the disabled prop from adjusting opacity by default to adjusting the color to `text-xweak` -- in the case of these snapshot changes it indicates a Button that is disabled in the test, and we are seeing that change as expected (opacity isn't changed, but now color is affected).

However, I'm glad you flagged this case because `text-xweak` (the disabled button color) is the same color as the default carousel buttons (meaning that when a carousel arrow button is disabled, it looks no different from the default). Let me touch base with @L0ZZI or @gregfuruiye on what we want to do since the conclusion from last Friday's call was to use text-xweak as the disabled color.

Reply:
Upon looking back at the designs, it appears that we want to use the following in light and dark modes:
- light mode: ['text-xweak'].dark
- dark mode: ['text-xweak'].light

Essentially, we reverse the light and dark. I just need to figure out how to implement that.

Reply:
An update here. The colors for the disabled have been updated for grommet-theme-hpe and would not need the above implementation. However, a discussion needs to occur about what these default values would be for base.js.